kiwi/builder/disk.py

```diff
@@ -863,12 +863,12 @@ def _write_generic_fstab(self, device_map, setup):
             device_map['root'].get_device(), '/',
             custom_root_mount_args, fs_check_interval
         )
-        if 'spare' in device_map and \
```

Comment:
I am surprised this is failing... wasn't it a hashmap device_map? Anyway I am absolutely fine with the alternative

Reply:
Only the part that accessed the dictionary with ```if 'swap' in device_map``` was failing because swap doesn't have to exist in the dict. If you access the dict that way python fails if the key does not exist. So actually I would have only needed to change that one line but it felt odd to me to access only the swap key via the get() method and so I changed it completely for consistency. Does this makes sense ?

Reply:
mmm I am not that sure, in my prompt the following

```python
map = {'swap': 'somevalue', 'root': 'someothervalue'}

if 'swap' in map:
  print('swap found')
else:
  print('swap not found')

if 'home' in map:
  print('home found')
else:
  print('home not found')
```

returns

```
swap found
home not found
```
 Anyway I like the way you coded it, I also prefer it :)

Reply:
Hmm, you are completely right, so what stupid things am I doing here ? There must be another reason, damn. At least that access method is considered the preferred solution, so I did not make it worse :)